### ISSUE TITLE
Lang="en"

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#210]
Each page must have a lang attribute on its html element. Should be lang="en-US"

**Comment by philipashlock** [comment]
This was already set as lang="en" but I see no reason for it not to be lang="en-US" 

Looks like there might be some issues with roots.io as noted here: https://github.com/roots/roots/issues/37#issuecomment-30553601

Fixed in 85ef307b44098e7a3ec80380cb4a24d06272edfc
